lib/reports/actioned-tasks/index.js

```diff
@@ -60,16 +61,24 @@ module.exports = ({ db, flow, logger, query: params }) => {
     let isOutstanding = false;
     let submitToActionDiff;
     let assignToActionDiff;
-
+    let resubmittedDiffs = [];
     task.activity
+      .filter(Boolean)
```

Comment:
When I run this against the seed data it seems to return rows with an activity of `[null]`. Not sure how it worked at all before.